src/Cms/File.php

```diff
@@ -248,12 +248,16 @@ public function dragText(string $type = null, bool $absolute = false): string
         if ($type === 'markdown') {
             if ($this->type() === 'image') {
                 return '![' . $this->alt() . '](' . $url . ')';
+            } elseif ($this->type() === 'video') {
+                return '[![ALT TEXT](IMAGE URL)](VIDEO URL)';
```

Comment:
Any idea about that?

Reply:
Hm, this is tricky. There's no video tag in markdown. We could only inject an html video tag instead.

Reply:
@bastianallgeier You mean like following. 
So actually rendered `video` tag in directly panel text editor?

**Write output to editor;**

If local or remote video

    <figure class="video"><video controls><source src="LOCAL-REMOTE-VIDEO-URL" type="video/mp4"></video></figure>

If youtube or vimeo video
    
    <figure class="video"><iframe allowfullscreen src="YOUTUBE-VIMEO-URL"></iframe></figure>

Reply:
Or should we just link to the file (local video) or video url (youtube, vimeo, remote video)?

Reply:
It's definitely not ideal. Another option could be to add a simple markdown link to the video instead. I don't know what makes more sense from a user perspective. All HTML is quite annoying in the context of markdown, but the link is just a very bad compromise either. Maybe keep it simple for this first implementation and only render the markdown link. Then we wait for feedback.

Reply:
I would go for the link. There is no video tag in Markdown that's the reality. Putting a lot of HTML in it should be a conscious decision by the user